apps/bfd-pipeline/bfd-pipeline-ccw-rif/src/main/java/gov/cms/bfd/pipeline/ccw/rif/extract/s3/S3Utilities.java

```diff
@@ -14,9 +15,19 @@ public final class S3Utilities {
    * Create an S3 client.
    *
    * @param options the {@link ExtractionOptions} to use
-   * @return the {@link AmazonS3} client to use
+   * @return the {@link S3Client} client to use
    */
-  public static AmazonS3 createS3Client(ExtractionOptions options) {
+  public static S3Client createS3Client(ExtractionOptions options) {
     return SharedS3Utilities.createS3Client(options.getS3Region());
   }
+
+  /**
+   * Create an Async S3 client.
+   *
+   * @param options the {@link ExtractionOptions} to use
+   * @return the {@link S3AsyncClient} client to use
+   */
+  public static S3AsyncClient createS3AsyncClient(ExtractionOptions options) {
+    return SharedS3Utilities.createS3AsyncClient(options.getS3Region());
+  }
```

Comment:
Why not just inline these and remove this class?  It appears to be obsolete to me.